README.md

```diff
@@ -41,6 +41,14 @@ truncate -s 1024G /tmp/disk.img
 sudo losetup -f /tmp/disk.img --show
 ```
 
+###### MetaPartition
+A meta partition is a small (~10MiB) partition that needs to be created on the disk, so as store the disk identification
+information. In device localpv, the partition name of the meta partition is used to identify the disk. A partition
+should follow the below criteria, so that openebs identifies it as a meta partition and use it for disk identification
+1. The partition number should be 1 (`ID_PART_ENTRY_NUMBER=1`)
+2. The partition should not be formatted with any filesystem
```

Comment:
do we verify that there is no filesystem or partitions does not have any flags set.

Reply:
Nope. We are not checking for filesystem and flags, only partition number is checked. cc: @praveengt Do we need to add this check?

Reply:
https://github.com/openebs/device-localpv/blob/ca58e5841c7a1a3ed74cb8e0913a98a6e678ad33/pkg/device/device-util.go#L411
Checking the column number to 5, will work only if flags and FS type did not exist

Reply:
Thank you. Didnt know about that.